.github/workflows/tests.yml

```diff
@@ -53,8 +53,10 @@ jobs:
         env:
           LV2_PATH: $GITHUB_WORKSPACE/test-blocks/build/
         run: |
+          mkdir test-presets
```

Comment:
this seems like the wrong place for it, the tests wont work without this dir, so best to create it inside the c++ tests code